src/goto-instrument/goto_instrument_parse_options.h

```diff
@@ -90,6 +90,11 @@ Author: Daniel Kroening, kroening@kroening.com
   "(show-threaded)(list-calls-args)" \
   "(undefined-function-is-assume-false)" \
   "(remove-function-body):"\
+  "(aggressive-slice)" \
+  "(aggressive-slice-call-depth):" \
+  "(aggressive-slice-preserve-function):" \
+  "(aggressive-slice-preserve-functions-containing):" \
+  "(aggressive-slice-preserve-all-direct-paths)" \
```

Comment:
The prevalent trend for these seems to be as an OPT_ macro, along with the coresponding help macro.

Reply:
I think this comment should be addressed.